xbmc/storage/MediaManager.cpp

```diff
@@ -626,19 +622,16 @@ void CMediaManager::EjectTray( const bool bEject, const char cDriveLetter )
 void CMediaManager::CloseTray(const char cDriveLetter)
 {
 #ifdef HAS_DVD_DRIVE
-#if defined(TARGET_DARWIN)
-  // FIXME...
-#elif defined(TARGET_FREEBSD)
-  // NYI
-#elif defined(TARGET_POSIX)
+#if defined(TARGET_POSIX)
+  std::shared_ptr<CLibcdio> c_cdio = CLibcdio::GetInstance();
```

Comment:
silly question: how is `CMediaManager::CloseTray` invoked? when i eject the disk the main menu item disappears, so no chance to reload it via gui?!

Reply:
yup, quite confusing. but i got it now